File: 2864-maximum-odd-binary-number/2864-maximum-odd-binary-number.py
class Solution:
    def maximumOddBinaryNumber(self, s: str) -> str:
        ones_count = s.count('1')
        zeros_count = s.count('0')
        
        if ones_count > 1:
            result = '1' * (ones_count - 1) + '0' * zeros_count + '1'
        elif ones_count == 1:
            result = '0' * zeros_count + '1'
        else:
            result = s
        
        return result
  

File: 2864-maximum-odd-binary-number/2864-maximum-odd-binary-number.go
func maximumOddBinaryNumber(s string) string {
    ones_count := 0
    zeros_count := 0
    for i := 0; i < len(s); i++ {
        if s[i] == '1'{
            ones_count++
        } else {
            zeros_count++
        }
    }
    return strings.Repeat("1", ones_count-1) + strings.Repeat("0", zeros_count) + "1"
}